### PR TITLE
fix: issue with fee UI react rendering loop

### PR DESCRIPTION
**Pull request by jrwbabylonlab**

### CODE REVIEW

File: src/utils/isStakingSignReady.ts [hunk 1]
```diff
@@ -39,6 +40,11 @@ export const isStakingSignReady = (
       isReady: false,
       reason: "Please enter a valid staking period",
     };
+  } else if (stakingFeeSat == 0) {
+    return {
+      isReady: false,
+      reason: "Not enough funds to cover fees for staking",
```
Comment (vitsalis) [inline src/utils/isStakingSignReady.ts:46]: Let's add a comment here as the `stakingFeeSat == 0` does not explain very well the situation behind the message.

**Review by vitsalis — APPROVED** [review]
LGTM. Let's have some documentation behind the decision.

**Review by jeremy-babylonlabs — APPROVED** [review]
lgtm, tested on chrome

File: src/utils/isStakingSignReady.ts [hunk 2]
```diff
@@ -39,6 +40,11 @@ export const isStakingSignReady = (
       isReady: false,
       reason: "Please enter a valid staking period",
     };
+  } else if (stakingFeeSat == 0) {
```
Comment (gbarkhatov) [inline src/utils/isStakingSignReady.ts:43]: Should we check for strict equals?

**Review by gbarkhatov — APPROVED** [review]

**Pull request merged**